app/dashboard/router.py

```diff
@@ -267,6 +269,16 @@ def get_queryset(self):
             statuses = self.request.query_params.get('status__in').split(',')
             queryset = queryset.filter(idx_status__in=statuses)
 
+        applicants = self.request.query_params.get('applicants')
```

Comment:
did you guys do any testing on performance here?  this is a multi table JOIN between bounty + interest tables, and it increases the load time for the API on the explorer from 500ms to 10s+

Reply:
note the diff between the `applicants=0` calls [here](http://bits.owocki.com/69cbd5ff245e/Image%202019-07-10%20at%201.44.09%20PM.png) vs the ones that dont have that call in them...

Reply:
the explorer page defaults to applicants=0, thus making most explorer lookups 10+s

Reply:
@owocki I will fix it